cmd/jb/main_test.go

```diff
@@ -64,12 +64,12 @@ func TestParseDependency(t *testing.T) {
 		},
 		{
 			name: "SSH",
-			path: "git+ssh://git@github.com/jsonnet-bundler/jsonnet-bundler.git",
+			path: "git+ssh://git@example.com/jsonnet-bundler/jsonnet-bundler.git",
 			want: &deps.Dependency{
 				Source: deps.Source{
 					GitSource: &deps.Git{
 						Scheme: deps.GitSchemeSSH,
-						Host:   "github.com",
+						Host:   "example.com",
 						User:   "jsonnet-bundler",
 						Repo:   "jsonnet-bundler",
 						Subdir: "",
```

Comment:
I'd still like to keep one test with github.com, simply cause I've seen that 99% of imports are actually Github.

Reply:
Oops, I thought I had left one in there - much agreed.